modules/database/main.tf

```diff
@@ -11,15 +11,27 @@ resource "azurerm_postgresql_flexible_server" "postgres" {
   administrator_login    = var.database_user
   administrator_password = var.password
 
-  storage_mb = 32768
+  storage_mb = var.storage_mb
   sku_name   = var.sku_name
 
   backup_retention_days = 7
 
+  lifecycle {
+    ignore_changes = [
+      zone
+    ]
+  }
+
```

Comment:
@kay-kim adding this to fix the zone error that you were seeing! Thank you for reporting it!

The fix is the same as the one used here:

https://github.com/wandb/terraform-azurerm-wandb/blob/437c895a082965abee21bc975dcce5e4530819e6/modules/database/main.tf#L48-L54

Reply:
Will try out now! ❤️